src/app/components/frontendService/FrontendServiceView.jsx

```diff
@@ -59,7 +59,9 @@ const FrontendServiceView = ({
       />
       <div className="frontend-service-main-view wrapper">
         {serviceLoaded ? (
-          <IFrame src={serviceUrl} width="100%" height="100%" />
+          service.active && (
+            <IFrame src={serviceUrl} width="100%" height="100%" />
+          )
         ) : (
           <Spinner loading={true} />
```

Comment:
Ich würde hier eine Fallbackkomponente anbieten, falls der Zielservice nicht existiert oder deaktiviert ist statt einem unendlichen Spinner.

Reply:
Ich habe einen Fallback eingebaut falls ein Service nicht existiert oder inaktiv ist. Außerdem habe ich die FrontendServiceView etwas refactored, weil der Spinner an dieser Stelle gar keinen Sinn gemacht hat.
(FrontendServices werden zum Start der App in den Store geladen, sind also zu diesem Zeitpunkt bereits vorhanden).